DockerDeployment.md

```diff
@@ -16,3 +16,73 @@ This method of deployment is used under the hood for:
 * [LunaNode Web Deployment](LunaNodeWebDeployment.md) *
 * [Azure Deployment](AzureDeployment.md) *
 * [Google Cloud Deployment](GoogleCloudDeployment.md) *
+
+## Hardening your Linux VPS
+
+Securing your BTCPay Server istance is important to protect your and your customers' data from the hands of hackers.
+It would take a whole chapter to explain best security practices; for this reason, we will simply explain the basic hardening.
+If you want to deepen this topic, you can follow [this tutorial](https://www.cyberciti.biz/tips/linux-security.html).
+
+###  Use strong passwords
```

Comment:
Better suggest here to use a [ssh-key-verifcation](https://www.digitalocean.com/community/tutorials/how-to-configure-ssh-key-based-authentication-on-a-linux-server) since this is something advanced users will be performing.

Reply:
Initially I had included the guide on how to disable password access and instead use keys.
I can reinsert it again if you want

Reply:
I think keys are much better suggestions, considering the section is called "hardening".

Reply:
If on the one hand safety increases, on the other it increases risks.

I think that for the average user this information is enough, authentication by keys I see it more suitable for more experienced users (who already know these things).

Do not you think?

Reply:
How about we just link to it and mentioned it as a good practice?